scheduler/src/main/java/org/apache/falcon/execution/SchedulerUtil.java

```diff
@@ -27,6 +28,14 @@
 
     private static final long MINUTE_IN_MS = 60 * 1000L;
     private static final long HOUR_IN_MS = 60 * MINUTE_IN_MS;
+    public static final String MINUTELY_PROCESS_FREQUENCY_POLLING_IN_MILLIS =
+            "falcon.scheduler.minutely.process.polling.frequency.millis";
```

Comment:
This seems too expanded. having multiple polling frequencies for each frequency may not be a good idea.

I feel falcon should start considering polling frequency from user as user is more aware on the input data availability. This we can create a jira and discuss. 
But for now can we have a common polling frequency for all. We might have to later deprecate these properties if we start having polling frequency in the process.